superset/translations/it/LC_MESSAGES/messages.po

```diff
@@ -2,1800 +2,803 @@
-msgstr ""
+msgstr "Cloud di Parole"
 
-#: superset/viz.py:466
+#: superset/viz.py:471
 msgid "Treemap"
-msgstr ""
+msgstr "Treemap"
 
-#: superset/viz.py:492
+#: superset/viz.py:497
 msgid "Calendar Heatmap"
-msgstr ""
+msgstr "Calendario di Intensità"
 
-#: superset/viz.py:550
+#: superset/viz.py:555
 msgid "Box Plot"
-msgstr ""
+msgstr "Box Plot"
 
-#: superset/viz.py:639
+#: superset/viz.py:644
 msgid "Bubble Chart"
-msgstr ""
+msgstr "Grafico a Bolle"
 
-#: superset/viz.py:687
+#: superset/viz.py:693
 msgid "Bullet Chart"
-msgstr ""
+msgstr "Grafico a Proiettile"
 
-#: superset/viz.py:736
+#: superset/viz.py:742
 msgid "Big Number with Trendline"
-msgstr ""
+msgstr "Numero Grande con Linea del Trend"
 
-#: superset/viz.py:765
+#: superset/viz.py:771
 msgid "Big Number"
-msgstr ""
+msgstr "Numero Grande"
 
-#: superset/viz.py:792
+#: superset/viz.py:798
 msgid "Time Series - Line Chart"
-msgstr ""
+msgstr "Serie Temporali - Grafico Lineare"
 
-#: superset/viz.py:919
+#: superset/viz.py:925
 msgid "Time Series - Dual Axis Line Chart"
-msgstr ""
+msgstr "Serie Temporali - Grafico Lineare ad Assi Duali"
 
-#: superset/viz.py:994
+#: superset/viz.py:1000
 msgid "Time Series - Bar Chart"
-msgstr ""
+msgstr "Serie Temporali - Grafico Barre"
 
-#: superset/viz.py:1002
+#: superset/viz.py:1008
 msgid "Time Series - Percent Change"
-msgstr ""
+msgstr "Serie Temporeali - Cambiamento Percentuale"
 
-#: superset/viz.py:1010
+#: superset/viz.py:1016
 msgid "Time Series - Stacked"
-msgstr ""
+msgstr "Serie Temporali - Stacked"
 
-#: superset/viz.py:1019
+#: superset/viz.py:1025
 msgid "Distribution - NVD3 - Pie Chart"
-msgstr ""
+msgstr "Distribuzione - NVD3 - Grafico Torta"
 
-#: superset/viz.py:1037
+#: superset/viz.py:1043
 msgid "Histogram"
-msgstr ""
+msgstr "Istogramma"
 
-#: superset/viz.py:1062
+#: superset/viz.py:1068
 msgid "Distribution - Bar Chart"
-msgstr ""
+msgstr "Distribuzione - Grafico Barre"
 
-#: superset/viz.py:1129
+#: superset/viz.py:1135
 msgid "Sunburst"
-msgstr ""
+msgstr "Sunburst"
 
-#: superset/viz.py:1162
+#: superset/viz.py:1168
 msgid "Sankey"
-msgstr ""
+msgstr "Sankey"
 
-#: superset/viz.py:1211
+#: superset/viz.py:1217
 msgid "Directed Force Layout"
-msgstr ""
+msgstr "Disposizione a Forza Diretta"
+
+#: superset/viz.py:1238
+msgid "Country Map"
+msgstr "Mappa della Nazione"
 
-#: superset/viz.py:1232
+#: superset/viz.py:1267
 msgid "World Map"
-msgstr ""
+msgstr "Mappa del Mondo"
 
-#: superset/viz.py:1282
+#: superset/viz.py:1317
 msgid "Filters"
-msgstr ""
+msgstr "Filtri"
 
-#: superset/viz.py:1317
+#: superset/viz.py:1352
 msgid "iFrame"
-msgstr ""
+msgstr "iFrame"
 
-#: superset/viz.py:1334
+#: superset/viz.py:1369
 msgid "Parallel Coordinates"
-msgstr ""
+msgstr "Coordinate Parallele"
 
-#: superset/viz.py:1359
+#: superset/viz.py:1394
 msgid "Heatmap"
-msgstr ""
+msgstr "Mappa di Intensità"
 
-#: superset/viz.py:1410
+#: superset/viz.py:1445
 msgid "Horizon Charts"
-msgstr ""
+msgstr "Grafici Orizzontali"
```

Comment:
Are this really `Orizzontali` and not `dell'orizzonte` if we are going to translate it literally?